### PR TITLE
[zh]Sync static-pod file

### PR DESCRIPTION
**Pull request by mengjiao-liu**
[zh]Sync static-pod file

### CODE REVIEW

File: content/zh/docs/tasks/configure-pod-container/static-pod.md [hunk 1]
```diff
@@ -287,19 +287,28 @@ The output might be something like:
...
+CONTAINER       IMAGE                                 CREATED           STATE      NAME    ATTEMPT    POD ID
+129fd7d382018   docker.io/library/nginx@sha256:...    11 minutes ago    Running    web     0          34533c6729106
 ```
 
+<!--
+`crictl` outputs the image URI and SHA-256 checksum. `NAME` will look more like:
+`docker.io/library/nginx@sha256:0d17b565c37bcbd895e9d92315a05c1c3c9a29f762b011a10c54a66cd53c9b31`.
+-->
+{{< note >}}
+`crictl` 会输出镜像 URI 和 SHA-256 校验和。 `名字` 看起来像：
```
Comment (mengjiao-liu) [inline content/zh/docs/tasks/configure-pod-container/static-pod.md:308]: `NAME` will look more like:
`docker.io/library/nginx@sha256:0d17b565c37bcbd895e9d92315a05c1c3c9a29f762b011a10c54a66cd53c9b31`.

我认为这个`NAME`应该指的是`IMAGE`列的值，所以直接译为了 `名字`，或者有什么更好的建议？

Comment (tengqm) [reply content/zh/docs/tasks/configure-pod-container/static-pod.md:308]: 不要翻译 NAME

Comment (mengjiao-liu) [reply content/zh/docs/tasks/configure-pod-container/static-pod.md]: Updated. Thanks.

**Pull request merged**